react/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "@vtex/tsconfig",
   "compilerOptions": {
+    "jsx": "react",
```

Comment:
Why is this change necessary? 🤔 
We're extending `@vtex/tsconfig`, so this should already be included